### PR TITLE
Major improvements to web watcher UX

### PR DESCRIPTION
**Pull request by ErikBjare**
- Added banner to the top of the popup
- Added field in popup to display if in testing mode
- Popup now shows when the last sync was performed
- Debug info is now in a <detail> tag
- Now uses the testing port (5666) when running in unpacked mode (fixes https://github.com/ActivityWatch/aw-watcher-web/issues/5)
- Now shows popup if failed to send heartbeat to the server

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
Last sync says "Invalid Date" before the first heartbeat goes through though

File: app/popup.html [hunk 1]
```diff
@@ -15,6 +15,19 @@
 
   <hr>
 
+  <div>
+    <a id="webui-link" href="http://localhost:5600/" target="_blank">
```
Comment (johan-bjareholt) [inline app/popup.html]: 5666 on testing

Comment (ErikBjare) [reply app/popup.html]: Just fixed in a new commit

File: app/popup_files/popup.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ function getCurrentTabs(callback) {
 
 function renderStatus() {
   chrome.storage.local.get(["lastSync"], function(obj) {
+    let lastSyncString = obj.lastSync ? new Date(obj.lastSync).toLocaleString() : "never";
```
Comment (ErikBjare) [inline app/popup_files/popup.js]: This should fix the "Invalid Date" string.

**Comment by ErikBjare** [comment]
I'll merge this then :+1:

**Pull request merged**